Source/Extensions/Blazorise.DataGrid/DataGridColumn.cs

```diff
@@ -192,6 +192,22 @@ internal string BuildFilterCellStyle()
             return sb.ToString().TrimStart( ' ', ';' );
         }
 
+        /// <summary>
+        /// Builds the Filter cell background.
+        /// IsFixedHeader feature needs to apply background color to columns. This makes sure to syncronize with the DataGrid header styling helpers.
+        /// </summary>
+        /// <returns></returns>
+        internal Background BuildFilterCellBackground()

```

Comment:
Make it a property `FilterCellBackground` and move it under the properties region. Same for `HeaderCellBackground`.